presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -100,6 +102,14 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return tableStatistics.build();
     }
 
+    private Set<String> filterPartitionColumnFrom(Map<String, ColumnHandle> tableColumns, List<HivePartition> hivePartitions)
```

Comment:
put this below usage, i..e below `getPartitionsStatistics` and make static

Reply:
done